packages/rune-games-cli/src/flows/Upload/Upload.tsx

```diff
@@ -22,11 +23,24 @@ export function Upload() {
   const { me } = useMe()
   const { games } = useGames({ skip: !me })
   const { myGames } = useMyGames({ games, devId: me?.devId })
+  const { release, name, confirm } = flags
 
   useEffect(() => {
     if (myGames && myGames.length > 1) setNeedConfirmation(true)
   }, [myGames])
 
+  useEffect(() => {
+    release && setReadyForRelease(release)
```

Comment:
Its possible that dev might want to both mark ready for release as yes or no. Is it possible to send --release false flag and would that set it as false?

Reply:
Done and done!, added that thanks